pure.zsh

```diff
@@ -227,9 +227,9 @@ prompt_pure_async_callback() {
 			prompt_pure_git_dirty=$output
 			prompt_pure_preprompt_render
 
-			# when prompt_pure_git_delay_dirty_check is set, the git info is displayed in a different color, this is why the
-			# prompt is rendered before the variable is (potentially) set
-			(( $exec_time > 2 )) && prompt_pure_git_delay_dirty_check=$EPOCHSECONDS
+			# when prompt_pure_git_last_dirty_check_timestamp is set, the git info is displayed in a different color,
+			# this is why the prompt is rendered before the variable is (potentially) set
```

Comment:
No, not in the sense that the comment is saying. I think something along these lines is more appropriate:
```sh
# When prompt_pure_git_delay_dirty_check_timestamp is set, the git info is displayed in a different color.
# To distinguish between a "fresh" and a "cached" result, the preprompt is rendered before setting this
# variable. Thus, only upon next rendering of the preprompt will the result appear in a different color.
```